### PR TITLE
Adds config mechanism for handlebars helpers and actor updates

### PR DESCRIPTION
**Pull request by eeyun**
This PR looks much bigger than it actually is.
It's inflated by find and replace that alters our localization. The pr adds a few functional updates. Namely it creates a config object that can be used to map against localization data to be used inside handlebars helpers. I also snuck in the start of actor update behaviors as base.js

Signed-off-by: Ian Henry <ianjhenry00@gmail.com>

### CODE REVIEW

File: scss/styles/conan2d20.css [hunk 1]
```diff
@@ -0,0 +1,590 @@
+/* ----------------------------------------- */
```
[the remark this answers is not in the document]
Comment (eeyun) [reply scss/styles/conan2d20.css:1]: Ah, i had been playing with reorganizing the scss files and things. I probably still will in a future pr. But i've fixed this commit.

File: scripts/config.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "bra": "CONAN.attributes.bra",
+    "coo": "CONAN.attributes.coo",
+    "int": "CONAN.attributes.int",
+    "per": "CONAN.attributes.per",
+    "wil": "CONAN.attributes.wil"
+  };
+
+/* -------------------------------------------- */
+/**
+ * The set of skill which can be trained in D&D5e
```
[the remark this answers is not in the document]
Comment (eeyun) [reply scripts/config.js:29]: Ah this is a bad copy paste from another file. In this case there is no logic here - this is just a variable declaration. However there is attribution elsewhere.

File: conan2d20.js [hunk 3]
```diff
@@ -27,6 +28,7 @@ Hooks.once('init', async function() {
    */
 
   // Define custom Entity classes
+  CONFIG.CONAN = CONAN;
```
[the remark this answers is not in the document]
Comment (eeyun) [reply conan2d20.js:31]: In other examples i've seen the config class is made all caps, i just assumed it was a community convention.

[the remark this answers is not in the document]
Comment (jeromelavoie) [reply conan2d20.js:31]: Probably

**Pull request merged**